Resources/Prototypes/Catalog/Research/technologies.yml

```diff
@@ -121,6 +121,7 @@
     - VaccinatorMachineCircuitboard
     - DiagnoserMachineCircuitboard
     - HandheldCrewMonitor
+    - StasisBedMachineCircuitboard
```

Comment:
this needs to be a much rarer tech imo, and stasis beds should be fairly expensive to construct. ideally they're one of the rarest machines that medical can get, to facilitate there being an actual economy with regards to who gets to use them in triage situations